roles/openshift_logging_fluentd/templates/fluentd.j2

```diff
@@ -168,6 +168,16 @@ spec:
           value: "{{ openshift_logging_fluentd_remote_syslog_payload_key }}"
 {% endif %}
 
+{% if audit_container_engine is defined %}
```

Comment:
@richm thoughts on if this is good enough or if we should run it through the bool filter as well?

Reply:
we should validate that the value is a bool value

Reply:
In that case we should consider using `bool` filter in all other cases.
It looks that the filter emits `False` for undefined variables.
```
>>> from ansible.plugins.filter.core import to_bool as bool
>>> from jinja2 import Template
>>> Template('').environment.filters['bool'] = bool
>>> tmpl = Template('{{ var | bool }}')
>>> print tmpl.render()
False
>>> print tmpl.render(var=False)
False
>>> print tmpl.render(var=True)
True
>>> print tmpl.render(var=123)
False
>>> print tmpl.render(var='abc')
False
>>> print tmpl.render(var='')
False
```

Reply:
Ok - then I'll leave that up to @ewolinetz and @sdodson to determine what is proper openshift-ansible syntax for this.

Reply:
@ewolinetz  I'm gonna rely on the `bool` filter. If the var is undefined, the filter will set it to `False`.
That way I process the variable only once in `tasks/main.yaml` instead of in all occurrences in `templates/fluentd.j2`

Reply:
@josefkarasek you'll likely want to do something like `{{ variable | default(False) | bool }}` I'm not sure that `{{ undefined_var | bool }}` will work with the actual `template` module.